wgpu-types/src/lib.rs

```diff
@@ -0,0 +1,506 @@
+use std::{io, slice};
```

Comment:
let's have a license header like we do in `wgpu` source files, so that we don't have a problem vendoring this in Gecko